service/src/test/java/datawave/microservice/accumulo/lookup/LookupServiceAuditDisabledTest.java

```diff
@@ -186,23 +180,23 @@ public void testLookupBeginEndSubset() throws Exception {
         
         String rowid = "row1";
         LookupResponse response = doLookup(defaultUserDetails, path(testTableName + "/" + rowid), queryString);
-        assertEquals("Lookup should have returned 4 entries", 4, response.getEntries().size());
-        assertTrue("First result should be cq3", ((DefaultKey) (response.getEntries().get(0).getKey())).getColQual().equals("cq3"));
-        assertTrue("Last result should be cq6", ((DefaultKey) (response.getEntries().get(3).getKey())).getColQual().equals("cq6"));
+        assertEquals(4, response.getEntries().size(), "Lookup should have returned 4 entries");
+        assertEquals("cq3", response.getEntries().get(0).getKey().getColQual(), "First result should be cq3");
+        assertTrue(response.getEntries().get(3).getKey().getColQual().equals("cq6"), "Last result should be cq6");
     }
     
     @Test
     public void testErrorOnBeginGreaterThanEnd() throws Exception {
-        expectedException.expect(HttpServerErrorException.class);
-        expectedException.expect(new TestHelper.StatusMatcher(500));
         //@formatter:off
         String queryString = String.join("&",
             "useAuthorizations=A,B,C,D,E,F,G,H,I",
             "columnVisibility=foo",
             LookupService.Parameter.CF + "=cf2",
             LookupService.Parameter.BEGIN_ENTRY + "=7",
             LookupService.Parameter.END_ENTRY + "=5");
-        doLookup(defaultUserDetails, path(testTableName + "/row2"), queryString);
+        Assertions.assertThrows(HttpServerErrorException.InternalServerError.class, () -> {
```

Comment:
static import for `assertThrows`

Reply:
This will be updated when I merge in Keith's accumulo 2.1 changes.

Reply:
got it